peerconnection.go

```diff
@@ -818,7 +818,16 @@ func (pc *PeerConnection) SetRemoteDescription(desc SessionDescription) error {
 				if err != nil {
 					return err
 				}
-				t = pc.newRTPTransceiver(receiver, nil, RTPTransceiverDirectionRecvonly, kind)
+
+				switch direction {
+				case RTPTransceiverDirectionSendonly, RTPTransceiverDirectionSendrecv:
+					t = pc.newRTPTransceiver(receiver, nil, RTPTransceiverDirectionRecvonly, kind)
+				case RTPTransceiverDirectionRecvonly, RTPTransceiverDirectionInactive:
+					fallthrough
```

Comment:
This could be omitted. For documentation a comment above `default:` would be enough.

Reply:
OK - removed the extra case and added a comment :+1: